io/disk/disk_info.py

```diff
@@ -191,20 +194,20 @@ def test(self):
         cmd = "blkid %s | cut -d '=' -f 2" % self.disk
         output = process.system_output(cmd, ignore_status=True,
                                        shell=True, sudo=True)
-        self.uuid = output.split('"')[1]
+        self.uuid = output.decode("utf-8").split('"')[1]
         self.log.info("Disk: %s UUID: %s", self.disk, self.uuid)
 
         # Verify mount point, filesystem type and UUID for each test variant
         output = process.system_output("lsblk -l %s" % self.disk,
                                        ignore_status=True, shell=True,
                                        sudo=True)
-        if self.dirs in output:
+        if self.dirs in output.decode("utf-8"):
```

Comment:
As decode is used in many places at `system_output` line, for uniformity better to maintain at same way through out the file.